.github/workflows/ci.yml

```diff
@@ -29,6 +29,7 @@ jobs:
       matrix:
         rails_version: ['5.2.6', '6.0.4.4', '6.1.4.4', '7.0.1', 'main']
         ruby_version: ['2.5', '2.6', '2.7', '3.0', '3.1']
+        use_global_output_buffer: ['true', 'false']
```

Comment:
Do you think there is a way to make the name of the checks clearer?

![image](https://user-images.githubusercontent.com/38524/159040702-bed22881-1ab1-4d00-854e-12365e4d2d7d.png)

ie. replacing `true` and `false` with something more meaningful

Reply:
Oh man, that's a great idea, thanks!